lib/assert.js

```diff
@@ -53,25 +53,21 @@ class AssertionError extends Error {
 		// Reserved for power-assert statements
 		this.statements = [];
 
-		if (opts.stack) {
-			this.stack = opts.stack;
+		if (opts.savedError && isError(opts.savedError)) {
```

Comment:
Can't we assume `savedError` is an error?

Reply:
You're right. We don't pass in anything else other than an error as savedError. Will change it